README.md

```diff
@@ -1,6 +1,17 @@
 # project-build-examples
 Sample projects to demonstrate project CI/CD features of the [`project-build-plugin`](http://axonivy.github.io/project-build-plugin/snapshot).
 
+## Versions and Branches
+You are currently on the master branch. Features shown here might still be under development and may not yet be released!
+
+If you are working with a LTS or LE version consider to switch to the corresponding branch:
```

Comment:
a link to our release-cycle would help here: so that customers can guess what the terms stand for: especially LE might not be clear to everyone.